libr/core/disasm.c

```diff
@@ -3510,7 +3510,7 @@ static void ds_print_ptr(RDisasmState *ds, int len, int idx) {
 		if (ds->immstr) {
 			char *str = r_str_from_ut64 (r_read_ble64 (&v, core->print->big_endian));
 			if (str && *str) {
-				const char *ptr = str;
+				const char *ptr = str + 1;
```

Comment:
First char is skipped ... why?

Reply:
Because thats how pascal strings work. Dirst char is the length of the string and then theres the string

Reply:
Only 1 byte for the length? If the length is 0xa (same value as newline), it's going to cause broken output.